mongodb/spring-cloud-starter-stream-source-mongodb/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1,3 @@
+configuration-properties.classes=org.springframework.cloud.stream.app.mongodb.source.MongoDBSourceProperties, \
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultUnlimited
```

Comment:
@markpollack , @ericbottard , shouldn't we add the `org.springframework.boot.autoconfigure.mongo.MongoProperties` here as well? Seems for me that is the most important part of properties. End-user will always specify `uri`, `database`, `credentials` etc. to his apps.
I see we don't have similar requirement for JDBC or Rabbit app. But isn't that like a simple omission?
Or am I missing anything?
Thanks